src/cascadia/TerminalSettingsEditor/Actions.h

```diff
@@ -93,10 +99,12 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         void OnNavigatedTo(const winrt::Windows::UI::Xaml::Navigation::NavigationEventArgs& e);
         Windows::UI::Xaml::Automation::Peers::AutomationPeer OnCreateAutomationPeer();
         void KeyChordEditor_PreviewKeyDown(Windows::Foundation::IInspectable const& sender, Windows::UI::Xaml::Input::KeyRoutedEventArgs const& e);
+        void OpenSettingsClick(const IInspectable& sender, const Windows::UI::Xaml::RoutedEventArgs& eventArgs);
 
         WINRT_CALLBACK(PropertyChanged, Windows::UI::Xaml::Data::PropertyChangedEventHandler);
         WINRT_PROPERTY(Editor::ActionsPageNavigationState, State, nullptr);
         WINRT_PROPERTY(Windows::Foundation::Collections::IObservableVector<Editor::KeyBindingViewModel>, KeyBindingList);
+        WINRT_PROPERTY(bool, ReadOnlyMode, !Feature_EditableActionsPage::IsEnabled());

```

Comment:
May not want to make this editable at runtime -- just add a getter that returns the right value instead of adding a member that stores it!

Reply:
Removed as a part of the "resurrect the 1.8 code as a completely separate XAML page" plan